tests/test_data_wrong_columns.csv

```diff
@@ -0,0 +1,3 @@
+name,address1
```

Comment:
This file is so small, you can easily build the dataframe from it in memory. No need to have it as a file, the test would run faster also.

e.g. 

```
# List1  
lst = [['tom', 25], ['krish', 30], 
       ['nick', 26], ['juli', 22]] 
    
df = pd.DataFrame(lst, columns =['Name', 'Age']) 
df 

```